### PR TITLE
docker: bump node and go versions

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
-FROM golang:1.12.4 as daemon
+FROM golang:1.12.9 as daemon
 
 COPY . /go/src
 WORKDIR /go/src
 RUN env GO111MODULE=on go build
 
-FROM node:10.15.3 as gui
+FROM node:10.16.3 as gui
```
Comment (chappjc) [inline Dockerfile]: Is 11.15 available?

Comment (chappjc) [reply Dockerfile]: As per matrix chat, current (12.x) is preferable for smoking out issues we'll eventually have to face.

Comment (dajohi) [reply Dockerfile]: travis builds the Dockerfile, which will test LTS.
travis also builds using the decred/dcrdata-golang-builder container which has current (12.x)

**Comment by chappjc** [comment]
So we're going to dump Docker and update to Go 1.13, right?

**Comment by chappjc** [comment]
Can I haz github actions?

**Pull request closed without merging**